ibis/expr/operations/relations.py

```diff
@@ -529,7 +529,7 @@ def __init__(self, table, metrics, by, having, predicates, sort_keys):
             sort_keys=sort_keys,
         )
 
-    @attribute
+    @property
     def schema(self):
```

Comment:
Why the simply `schema` properties are turned into attributes while a complicated one like this turned into a property?

Reply:
This was accidental. I agree, it should be an `attribute`.

Reply:
Brought back.